frontend/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@hookform/resolvers": "^1.3.6",
```

Comment:
why do we need this extra package?

Reply:
Actually  hookform/resolvers help in validating React Forms  and it support YUP library , so these two form best combination
in implementing Validation in React Forms